redhat-developer/app-services-ansible
# chore: increase stability of the create kafka acl module

To verify:

Checkout PR
- run `make setup` or at least run `pip install rhoas-sdks --force-reinstall` to make sure you're running the latest version of the rhoas python sdks (v0.3.1)
- run `make local-dev`
- run `ansible-playbook 'run.yml'`

The updated `run.yml` script here will create:
-  a new kafka (if you have quota, you may also need to change the name of the kafka if someone else runs this script first, to ensure a unique kafka name) *NOTE* this step may take a while as now the `create_kafka` module waits for the new kafka to be in a ready state before finishing
-  new service account 
- create an ACL and bind it to the newly created service account on your new kafka
- create 2 topics
  - one with no (API default) config 
  - one with custom config
